roles/keystone/tasks/main.yml

```diff
@@ -89,11 +107,19 @@
   notify: restart keystone services
   # we want this to always be changed so that it can notify the service restart
   tags: db-migrate
-
+  
+- name: enable keystone apache site
+  apache2_site: name=keystone state=enabled
+  notify:
+    - reload apache
+    
 - meta: flush_handlers
 
 - name: start keystone
   service: name=keystone state=started
+  
+- name: start keystone (apache)
+  service: name=apache2 state=started
```

Comment:
Yes, that is pretty much correct for keystone. You'd stop to the uwsgi proc and in that case apache would throw a 502.